src/IB/IIMethod.cpp

```diff
@@ -235,13 +235,84 @@ IIMethod::getFEDataManager(const unsigned int part) const
 } // getFEDataManager
 
 void
-IIMethod::registerDisconElemFamilyForJumps(const unsigned int part)
+IIMethod::registerDisconElemFamilyForTraction(const unsigned int part, 
+                                          libMesh::FEFamily fe_family,
+                                          libMesh::Order fe_order)
 {
     TBOX_ASSERT(!d_fe_equation_systems_initialized);
     TBOX_ASSERT(part < d_num_parts);
-    d_use_discon_elem_for_jumps[part] = true;
+    // The acceptable options are CONSTANT MONOMIAL,  FIRST order MONOMIAL, or FIRST order L2_LAGRANGE.
+    // NOTE: For now if separate orders are used for viscous vs. pressure jumps, the traction order 
+    // need to be the the lowest of the two.
```

Comment:
Can you move this documentation to the header file?

Reply:
Done.